lib/valueflow.cpp

```diff
@@ -106,16 +106,16 @@ static void bailoutInternal(TokenList *tokenlist, ErrorLogger *errorLogger, cons
         fileName = fileName.substr(p + 1);
     }
     ErrorLogger::ErrorMessage errmsg(callstack, tokenlist->getSourceFilePath(), Severity::debug,
-        std::string(fileName) + ":" + std::to_string(static_cast<unsigned long long>(line)) + ":" + std::string(function) + " bailout: " + what, "valueFlowBailout", false);
+        std::string(fileName) + ":" + MathLib::toString(static_cast<unsigned long long>(line)) + ":" + std::string(function) + " bailout: " + what, "valueFlowBailout", false);
     errorLogger->reportErr(errmsg);
 }
 
 #if (defined __cplusplus) && __cplusplus >= 201103L
-#define bailout(tokenlist, errorLogger, tok, what)	bailoutInternal(tokenlist, errorLogger, tok, what, __FILE__, __LINE__, __func__)
+#define bailout(tokenlist, errorLogger, tok, what)	bailoutInternal(tokenlist, errorLogger, tok, what, std::string(__FILE__), __LINE__, std::string(__func__))
```

Comment:
I would like that `std::string(__FILE__)` is replaced with `__FILE__`

Reply:
to me it looks weird that there is not a space before the `bailoutInternal`

Reply:
Removed temporary std::string objects for __FILE__ and __FUNCTION__.
Changed tabulator to space before bailoutInternal (that was an old habit of me).